calendar.js

```diff
@@ -135,11 +134,11 @@ function apiEventToFeedObject(event) {
 	}
 
 	var result = {
-		start: new Date(event.startDateTime),
-		end: new Date(event.endDateTime),
-		summary: prepend + (event.shortTitle || event.title || ('Tech@NYU Event')),
-		description: event.description || event.details,
-		url: event.rsvpUrl || ''
+		start: new Date(event.attributes.startDateTime),
+		end: new Date(event.attributes.endDateTime),
+		summary: prepend + (event.attributes.shortTitle || event.attributes.title || ('Tech@NYU Event')),
+		description: event.attributes.description || event.attributes.details,
+		url: event.attributes.rsvpUrl || ''
 	};
 
 	if (event.links && event.links.venue && event.links.venue.linkage) {
```

Comment:
The line below needs to be `attributes.address` rather than just `address`.

Also, the variable `venueIdsToVenueAddresses` should really be renamed to `venueIdsToVenues`, since it doesn't actually map the ids straight to addresses (which is a good thing, because we might want to include, e.g., the venue title in our feed item down the line).